src/picknik_ur_base_config/config/base_config.yaml

```diff
@@ -223,4 +223,4 @@ objectives:
   # [Required]
   waypoints_file:
     package_name: "picknik_ur_base_config"
-    relative_path: "waypoints/ur5e_waypoints.yaml"
+    relative_path: "waypoints/ur_waypoints.yaml"
```

Comment:
I worry about letting someone choose the UR model and giving them access to waypoints that were specific to a UR5e. What do you think about stripping this file back to a single pose that works on all models (like a zero pose)?